Playground/SwiftShortcutsTemplate.playgroundbook/Contents/Modules/ShortcutSupport.playgroundmodule/Sources/ShortcutPreviewViewController.swift

```diff
@@ -0,0 +1,130 @@
+            titleLabel.trailingAnchor.constraint(equalTo: button.trailingAnchor, constant: -8),
+            titleLabel.bottomAnchor.constraint(equalTo: button.bottomAnchor, constant: -6)
+        ])
+    }
+
+    @objc func share(_ sender: UIButton) {
+        do {
+            let fileManager = FileManager.default
+            let applicationSupport = try fileManager.url(
+                for: .applicationSupportDirectory,
+                in: .userDomainMask,
+                appropriateFor: nil,
+                create: false
+            )
+            let directoryURL = applicationSupport.appendingPathComponent("io.a2.swift-shortcuts", isDirectory: true)
+
+            // Remove the output directory if it already existed (clean up any old shortcuts)
+            if fileManager.fileExists(atPath: directoryURL.path) {
+                try fileManager.removeItem(at: directoryURL)
+            }
+
+            // Create the directory again
+            try fileManager.createDirectory(
+                at: directoryURL,
+                withIntermediateDirectories: true,
+                attributes: nil
+            )
+
+            // Build the shortcut and write it to disk
+            let shortcutURL = directoryURL.appendingPathComponent("\(name).shortcut", isDirectory: false)
+            let sortcutData = try shortcut.build()
+            try sortcutData.write(to: shortcutURL)
+
+            // Create an activity view controller pointing to the shortcut
+            let viewController = UIActivityViewController(
+                activityItems: [shortcutURL],
+                applicationActivities: nil
+            )
+
+            // Additionally configure iPad popover presentation if it's ever required
+            let controller = viewController.popoverPresentationController
+            controller?.sourceRect = sender.bounds
+            controller?.sourceView = sender
+
+            // Present the share sheet
+            present(viewController, animated: true, completion: nil)
+        } catch {
+            // Construct an error alert
+            let viewController = UIAlertController(
+                title: "Error Exporing",
+                message: error.localizedDescription,
+                preferredStyle: .alert
+            )
+
+            // Provide a dismiss action
+            viewController.addAction(
+                UIAlertAction(title: "Dismiss", style: .cancel, handler: nil)
+            )
+
+            // Present the alert controller
+            present(viewController, animated: true, completion: nil)
+        }
+    }
+
+    func dragInteraction(
+        _ interaction: UIDragInteraction,
+        itemsForBeginning session: UIDragSession
+    ) -> [UIDragItem] {
+        return [
+            UIDragItem(
+                itemProvider: NSItemProvider(
+                    object: ShortcutWrapper(shortcut)
+                )
+            )
+        ]
```

Comment:
Annoyingly this doesn't actually seem to work for me when I try to drag the shortcut into the Shortcuts.app 😕 
It does however work if I drag into the Files.app though so still worth keeping around. 

I'm not sure if it relates to me running iOS 14 or not though since I get the green "+" indicator in the Shortcuts app but nothing happens when I drop (if I try with a different drag item (like text), I don't even get the green "+")

I might end up filing a feedback about this after I dig a little deeper as it would be a nice to have

Reply:
Looks like the code here _should_ be working so i filled a Feedback about it... Might only be an iOS 14 bug still but worth keeping the code either way i think